sys/net/gnrc/link_layer/lwmac/lwmac.c

```diff
@@ -878,7 +870,16 @@ static void _lwmac_msg_handler(gnrc_netif_t *netif, msg_t *msg)
             lwmac_schedule_update(netif);
             break;
         }
-
+#if (GNRC_MAC_ENABLE_DUTYCYCLE_RECORD == 1)
+        case GNRC_MAC_TYPE_GET_DUTYCYCLE:
+            /* Output LWMAC's radio duty-cycle ratio */
+            uint64_t duty;
```

Comment:
If you declare a variable locally to this `case`, you have to protect the block within curly braces, otherwise it doesn't compile (your initial changes were perfectly fine!).

Reply:
Aha！~  Thanks for noticing me this! Get it back!